client/client.go

```diff
@@ -106,7 +106,7 @@ func (c *Client) RecursiveQuery(m *dns.Msg, tracer Tracer) (r *dns.Msg, rtt time
 	qname := m.Question[0].Name
 	qtype := m.Question[0].Qtype
 	zone := "."
-	for i := 1; i < 100; i++ {
+	for z := 1; z < 4; z++ {
```

Comment:
Feel like this change is unrelated to this fix right?

Reply:
ah yes, it just that the result the same either 100 or 4

Reply:
reverted

Reply:
Can you please also revert the i->z rename?